_extras/guide.md

```diff
@@ -167,6 +167,26 @@ particular set of files in `.gitignore`.
     repositories in the web and provides a web interface to interact with repos
     they host.
 
+*   Using personal access tokens to authenticate with GitHub from the command line
+    might be confusing for some learners as GitHub has only recently deprecated
+    password authentication. You can point advanced learners interested in the
+    rationale behind the decision by GitHub to [this blog post explaining it][github-access-tokens].
```

Comment:
`this blog post explaining it` is not a "good" link in terms of accessibility (see [What's wrong with using "click here" links](https://webaccess.berkeley.edu/ask-pecan/click-here)). I'd suggest something like the following:
```suggestion
    password authentication. You can point advanced learners interested to [the
    rationale behind the decision of deprecating passwords by GitHub][github-access-tokens].
```